setup.py

```diff
@@ -65,14 +64,8 @@ def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
 
-class PyTest(TestCommand):
-    user_options = []
-
-    def run(self):
-        import subprocess
-        errno = subprocess.call([sys.executable, '-m', 'pytest', 'tests'])
-        raise SystemExit(errno)
-
+NEEDS_PYTEST = {'pytest', 'test'}.intersection(sys.argv)
```

Comment:
How it works?

Reply:
this just checks for convenient argument at adds pytest-runner to setup_requires only if it's needed, meaning it won't install `pytest-runner` if you just do `sdist` or `bdist_wheel`

Reply:
But you never check the value, isn't it?

Reply:
Oops, it's intended to be checked on the next line, I'll fix this.

Reply:
@asvetlov done